src/typing_extensions.py

```diff
@@ -442,7 +439,7 @@ def _maybe_adjust_parameters(cls):
     """
     tvars = []
     if '__orig_bases__' in cls.__dict__:
-        tvars = typing._collect_type_vars(cls.__orig_bases__)
+        tvars = _collect_type_vars(cls.__orig_bases__)
```

Comment:
This change was required because `typing._collect_type_vars` doesn't exist on Python 3.11 (but `typing_extensions._collect_type_vars` does)